client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -105,32 +113,65 @@ class PublishingTools extends Component {
 		const formPending = this.isFormPending();
 
 		return (
-			<FormFieldset>
-				<SupportInfo
-					text={ translate(
-						'Allows you to publish new posts by sending an email to a special address.'
-					) }
-					link={
-						isAtomic
-							? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
-							: 'https://jetpack.com/support/post-by-email/'
-					}
-					privacyLink={ ! isAtomic }
-				/>
-				<JetpackModuleToggle
-					siteId={ selectedSiteId }
-					moduleSlug="post-by-email"
-					label={ translate( 'Publish posts by sending an email' ) }
-					disabled={ formPending || moduleUnavailable }
-				/>
+			<>
+				<FormFieldset>
+					<SupportInfo
+						text={ translate(
+							'Allows you to publish new posts by sending an email to a special address.'
+						) }
+						link={
+							isAtomic
+								? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
+								: 'https://jetpack.com/support/post-by-email/'
+						}
+						privacyLink={ ! isAtomic }
+					/>
+					<JetpackModuleToggle
+						siteId={ selectedSiteId }
+						moduleSlug="post-by-email"
+						label={ translate( 'Publish posts by sending an email' ) }
+						disabled={ formPending || moduleUnavailable }
+					/>
+
+					{ this.renderPostByEmailSettings() }
+				</FormFieldset>
+				<hr />
+			</>
+		);
+	}
+
+	renderPostByVoiceModule() {
+		return (
+			<>
+				<FormFieldset>
+					<PostByVoiceSetting />
+				</FormFieldset>
+				<hr />
+			</>
+		);
+	}
 
-				{ this.renderPostByEmailSettings() }
+	renderPressThisModule() {
+		return (
+			<FormFieldset>
+				<PressThis />
 			</FormFieldset>
 		);
 	}
 
 	render() {
-		const { selectedSiteId, translate } = this.props;
+		const { selectedSiteId, translate, siteIsJetpack, isAtomic } = this.props;
+
+		const renderPressThis =
+			config.isEnabled( 'press-this' ) &&
+			( siteIsJetpack || ( ! siteIsJetpack && ! this.isMobile() ) );
+		const renderPostByEmail = config.isEnabled( 'press-this' ) && siteIsJetpack;
```

Comment:
```suggestion
		const renderPressThis = config.isEnabled( 'press-this' ) && ! this.isMobile();
		const renderPostByEmail = siteIsJetpack;
```
I think this reflects the previous logic better. Whether or not this is a Jetpack site, we still want to render `Press This`.

Reply:
>I think this reflects the previous logic better.

Hm, I would not say so, previously we had two cases:
1) `siteIsJetpack && config.isEnabled( 'press-this' )` - rendered both "Press this" and "Post by Email"
2) `config.isEnabled( 'press-this' ) && ! this.isMobile() && ! siteIsJetpack` - rendered only "Press this"

So previously we had a case when nothing was rendered, it's "mobile + simple site" - `this.isMobile() && ! siteIsJetpack`. I suppose that there was some reason, since "Publishing tools" was duplicated. That why I wrote exactly this predicate to reflect previous logic completely the same way.

> const renderPostByEmail = siteIsJetpack;

Here is the same, I reflected the previous logic completely the same way. But I suppose that we can modify and avoid `config.isEnabled( 'press-this' )` here. I will remove.

Reply:
> So previously we had a case when nothing was rendered, it's "mobile + simple site" - this.isMobile() && ! siteIsJetpack. I suppose that there was some reason

I actually assume this was more of an oversight. Here's my reasoning: the instructions for the `Press This` bookmarklet say:

> Drag and drop the "Press This" button below to your bookmarks bar

Dragging it to the bookmarks bar only works on desktop, and that's why I think we have the `! this.isMobile()` condition. We use the exact same `PressThis` component on Simple and Atomic, so I don't see why this requirement would be any different between the two.

Ergo, I think we should follow my suggestion here after all.